platform/security/core/security-core-impl/src/main/java/ddf/security/assertion/impl/SecurityAssertionImpl.java

```diff
@@ -1657,6 +1517,12 @@ public void setNil(Boolean newNil) {
         public void setNil(XSBooleanValue newNil) {
 
         }
+
+        @Nonnull
```

Comment:
Doesn't this fail findbugs? Explicitly returning `null` from an `@Nonnull` annotated method?